go/interpreter/lfvm/opcode.go

```diff
@@ -447,3 +447,53 @@ func (o OpCode) HasArgument() bool {
 	}
 	return false
 }
+
+func (o OpCode) isSuperInstruction() bool {
```

Comment:
NICE. but should this then be a change in a follow-up to  #734 ? it seems like it is a more the domain of opcodes rework, rather than stack. Maybe this function and an extended `lfmv.isValid` should be in an intermediary PR after #734 and before this one? wdyt ?